TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Constants.java

```diff
@@ -41,34 +52,82 @@ public static abstract class DriveConstants {
         public static final double DEADZONE = 0.1;
     }
 
-    public static final int INCH_TO_TILE = 24;
-
-    public static final double SLIDE_PULLEY_CIRCUMFERENCE = 5.0*0.393701*Math.PI;
-
-    public static final double MOTOR_TICKS_PER_REVOLUTION = 751.8;
-
     public static abstract class IntakeConstants{
         public static final String HORIZONTAL_SLIDE_LEFT_MOTOR_NAME = "horizontalSlideLeftMotor";
         public static final String HORIZONTAL_SLIDE_RIGHT_MOTOR_NAME = "horizontalSlideRightMotor";
 
         public static final DcMotorSimple.Direction HORIZONTAL_SLIDE_LEFT_MOTOR_DIRECTION = DcMotorSimple.Direction.FORWARD;
-        public static final DcMotorSimple.Direction HORIZONTAL_SLIDE_RIGHT_MOTOR_DIRECTION = DcMotorSimple.Direction.FORWARD;
-        /*why both motor do not need changing direction is bcz gear problem, imagine O is motor and 0 is the gear:
-        someone in build team can explain,but it works
-         */
+        public static final DcMotorSimple.Direction HORIZONTAL_SLIDE_RIGHT_MOTOR_DIRECTION = DcMotorSimple.Direction.REVERSE;
```

Comment:
Reminder that we need to make sure that the direction are actually correct. This will happen with testing.